docker/ravendb-ubuntu/putdb.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+# It takes a certificate path as first argument and key path as 2nd
+
+if [[ $# == 0 ]]; then
```

Comment:
Please introduce `mode` variable and set it accordingly
```
[[ $# == 0 ]] && mode=unsecured || mode=secured
```

Then check what mode we're in. 

Other approach is to assign subsequent parameters to properly named vars and then test for empty/not empty with `-z` and `-n`.